testing/hardhat-deployment/dependency/dependency.sh

```
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: BUSL-1.1
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)